package.json

```diff
@@ -27,31 +28,30 @@
     "url": "https://github.com/iTowns/itowns2/issues"
   },
   "homepage": "https://itowns.github.io/",
+  "dependencies": {
```

Comment:
Yes, that's exactly what this PR is about: consuming the NPM module like any other library, with its own dependencies, rather than (necessarily) as a bundle (and then bundle things yourself using webpack or browserify; and properly deduplicate those dependencies used by your own code and by iTowns)